backend/app-kubernetes-service/src/main/kotlin/services/UtilizationService.kt

```diff
@@ -0,0 +1,141 @@
+package dk.sdu.cloud.app.kubernetes.services
+
+import com.github.jasync.sql.db.util.length
+import dk.sdu.cloud.app.kubernetes.services.volcano.VolcanoJob
+import dk.sdu.cloud.app.kubernetes.services.volcano.volcanoJob
+import dk.sdu.cloud.app.orchestrator.api.CpuAndMemory
+import dk.sdu.cloud.app.orchestrator.api.JobUtilization
+import dk.sdu.cloud.service.k8.*
+
+/**
+ * A service responsible for fetching utilization information
+ */
+class UtilizationService(
+    private val k8: K8Dependencies
+) {
+    suspend fun allocatable(): CpuAndMemory {
+        val namespace = kotlin.runCatching {
+            k8.client.getResource<Namespace>(
+                KubernetesResources.namespaces.withName("app-kubernetes")
+            )
+        }.getOrThrow()
+
+        val computeAnnotation = namespace.metadata?.annotations?.get("scheduler.alpha.kubernetes.io/node-selector").toString()
+        val nodes = if (computeAnnotation == "ucloud-compute=true") {
+            kotlin.runCatching {
+                k8.client.listResources<Node>(
+                    KubernetesResources.node.withNamespace(NAMESPACE_ANY)
+                ).items.filter { node ->
+                    node.metadata?.labels?.get("ucloud-compute") == "true"
+                }
+            }
+        } else {
+            kotlin.runCatching {
+                k8.client.listResources(
+                    KubernetesResources.node.withNamespace(NAMESPACE_ANY)
+                )
+            }
+        }
+
+        val nodeAllocatableCpu = nodes.getOrThrow().sumOf { node ->
+            node.status?.allocatable?.cpu?.toDouble() ?: 0.0
+        }
+
+        val nodeAllocatableMemory = nodes.getOrThrow().sumOf { node ->
+            memoryStringToBytes(node.status?.allocatable?.memory)
+        }
+
+        return CpuAndMemory(nodeAllocatableCpu, nodeAllocatableMemory)
+    }
+
+    suspend fun used(): CpuAndMemory {
+        val jobs = runCatching {
```

Comment:
This contains all jobs, including those in the queue. It should only count those which are actually running.